package.json

```diff
@@ -48,6 +48,8 @@
     "@jupyterlab/application": "^4.0.0-alpha.7",
     "@jupyterlab/apputils": "^4.0.0-alpha.7",
     "@jupyterlab/docregistry": "^4.0.0-alpha.7",
+    "@jupyterlab/filebrowser": "^3.3.4",
+    "@jupyterlab/launcher": "^3.3.4",
```

Comment:
The error you mentioned, is because you are using packages from JupyterLab 3.3 and building them with TypeScript 4.5.
Wolf fixed it in https://github.com/QuantStack/jupyterlab-blockly/pull/17 to be able to create the extension for JupyterLab 3. If you rebase with the master branch this will be fixed.